qa/suites/fs/mgr/per_module_finisher_thread/tasks/per_module_finisher_stats.yaml

```diff
@@ -0,0 +1,18 @@
+tasks:
+  - check-counter:
+      counters:
+        mgr:
+            - "finisher-balancer.complete_latency.avgcount"
+            - "finisher-cmd_finisher.complete_latency.avgcount"
+            - "finisher-crash.complete_latency.avgcount"
+            - "finisher-devicehealth.complete_latency.avgcount"
+            - "finisher-iostat.complete_latency.avgcount"
+            - "finisher-pg_autoscaler.complete_latency.avgcount"
+            - "finisher-progress.complete_latency.avgcount"
+            - "finisher-status.complete_latency.avgcount"
+            - "finisher-telemetry.complete_latency.avgcount"
+            - "finisher-volumes.complete_latency.avgcount"
```

Comment:
Since this is in the `fs` suite, I think we should just make it specific to the volumes plugin (i.e., remove these other modules).

Reply:
You can also specify a minimum if you know the exact number of commands that will be queued. I'd also suggest enhancing `check_counter.py` to allow specifying a max. Then you can confirm the queue is drained in this test (max == 0).

Reply:
youc an put the generic test in the `rados` suite

Reply:
@batrick Done. PTAL